### ISSUE TITLE
(tiny) Feature proposal: add cards to Wish list only if they are NOT in your collection

### ISSUE THREAD

**Issue by Naereen** [moxfield/moxfield-public#58]
Hi guys!
First of all, all my sincerest congratulations for the amazing work you did and keep doing on Moxfield.com, it's a very great website and I love it! I'm even supporting you on Patreon!!

I'm working on some decks, and I use the awesome "add to your collection" feature to track cards I already own.
However, I haven't found any way to filter the cards I don't own, to export them to a CardMarket.com wish list for instance.
I can add the entire deck to my wish list, then delete manually every card that I already own in collection, but it would be tedious. This could easily be automated from the website part, I think.

So long story short, there I am submitting you team this idea of a new feature!

Thanks in advance if it can be implemented, and cheers from France!
Lilian @Naereen
